docs/API.md

```diff
@@ -136,13 +136,15 @@ Each integration also has its own list of default tags. These tags get automatic
 | http.url         | The complete URL of the request.                          |
 | http.method      | The HTTP method of the request.                           |
 | http.status_code | The HTTP status code of the response.                     |
+| http.headers.*   | A recorded HTTP header.                                   |
 
 <h5 id="express-config">Configuration Options</h5>
 
 | Option           | Default                   | Description                            |
 |------------------|---------------------------|----------------------------------------|
 | service          | *Service name of the app* | The service name for this integration. |
 | validateStatus   | `code => code < 500`      | Callback function to determine if there was an error. It should take a status code as its only parameter and return `true` for success or `false` for errors. |
+| recordHeaders    | `[]`                      | An array of headers to include in the span metadata. |
```

Comment:
Me again, your favorite nitpicker :) I think simply `headers` would make more sense to avoid a verb since this is not a function nor a boolean value.

Reply:
Hah. I appreciate it, naming is hard 👍